Mlem/App/Views/Shared/ErrorView.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  Error View.swift
+//  Mlem
+//
+//  Created by David Bureš on 19.06.2022.
+//
+
+import Combine
+import MlemMiddleware
+import SwiftUI
+import UniformTypeIdentifiers
+
+struct ErrorView: View {
+    @AppStorage("developerMode") var developerMode: Bool = false
+    
+    @State var errorDetails: ErrorDetails
+    
+    @State private var showingFullError: Bool = false
+    @State private var refreshInProgress: Bool = false
+    
+    var timer = Timer.publish(every: 1, tolerance: 0.5, on: .main, in: .common)
+        .autoconnect()
+    
+    init(_ errorDetails: ErrorDetails) {
+        var errorDetails = errorDetails
```

Comment:
Why is this declared as a variable before state init?

(also this line throws a warning suggesting to make it a `let`)

Reply:
It shouldn't be; I'll remove it. I just copy-pasted `ErrorView` from 1.0 lol